### PR TITLE
Views

### PR DESCRIPTION
**Pull request by Stephen-Adom**
# Blog app - Views
In this pull request, the following tasks were implemented:
- Used methods implemented in the models
- Applied styling using tailwind css in designing the wireframe

### CODE REVIEW

File: app/views/posts/_post-card.html.erb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<div class="post-card border border-gray-300 rounded-md p-5 mb-5 bg-gray-100">
+    <h4 class="font-semibold text-lg mb-5"><%= post.title %> </h4>
+    <p>
+        <%= post.text %>
+    </p>
+
+    <p class="text-right text-sm mt-3">Comments: 10 Likes: 4</p>
```
Comment (KDlamini) [inline app/views/posts/_post-card.html.erb]: Kindly make sure that you display the number of comments and likes based on the comments_counter and likes_counter. Here you are displaying fixed values for comments and likes.

**Review by KDlamini — CHANGES_REQUESTED** [review]
### Hi @Stephen-Adom 🙋‍♀️,  

Great work so far! 💯 
You have done a great job creating views for the Blog app. 🎊 🥳 
There are some issues that you still need to work on to improve your project according to the requirements. You are almost there! 💪🏾 

## Highlights 🌞

✔️  Users controller `#index` action renders all users to it's corresponding Views index page
✔️  The `#recent_post` method renders the user's 3 most recent posts.
✔️  `See all Post` link opens the user's post index as displayed on the wireframe
✔️  The `post.recent_comments` renders 5 most recent comments
✔️  Post show page displays post with all comments

## Required Changes ♻️
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/views/posts/_post-card.html.erb (hunk 1 above)
Comment (Stephen-Adom) [reply app/views/posts/_post-card.html.erb]: Hello @KDlamini, thank you for the reminder. I forgot about that section. I will implement the changes

**Review by Ghiftee — APPROVED** [review]
## STATUS: APPROVED ♻️♻️
Hi @Stephen-Adom, 

To highlight 👍🏽 
✔️ App works as expected
✔️ Errors are fixed

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀  
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**